railties/lib/rails/generators/rails/app/templates/config/initializers/handle_cookies_without_metadata.rb.tt

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
+
+# Signed and encrypted cookies now have metadata associated with it.
+# If you don't want your existing cookies to be rejected immediately you need to specify a window
+# for how long those cookies should be honored as a Time or ActiveSupport::Duration object.
+
+# Also, setting this value to nil or to a date in the past rejects the existing cookies upfront.
+Rails.application.config.action_dispatch.fail_cookies_without_metadata_after = <%= 1.month.from_now %>
```

Comment:
@mikeycgto @kaspth @guilleiguaran
Travis CI's GEM = aj:integration fails with a syntax error because of the output of `<%= 1.month.from_now %>`. Should I convert `<%= 1.month.from_now %>` into a string or any other format? Or am I missing anything?